js/modules/offline-exporting.src.js

```diff
@@ -6,11 +6,34 @@
  * License: www.highcharts.com/license
  */
 
+/**
+ * Download options for offline exporting.
+ *
+ * @interface Highcharts.OfflineExportingOptionsObject
```

Comment:
I don't think we need a specific options object for offline exporting. The `libURL` option is the only thing missing in online, while `scale` already exists in online. We should rather add these to the `Highcharts.ExportingOptionsObject`.